assets/src/edit-story/components/colorPicker/useKeyDeleteStop.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { useKeyDownEffect } from '../keyboard';
+
+function useKeyDeleteStop(ref, onDelete) {
+  useKeyDownEffect(
+    ref,
+    ['del', 'backspace'],
```

Comment:
Don't we support `delete` as a special case for this? See `keys.js`.

Reply:
Ah yes, fixed in bc05c519.

Reply:
Ah yes, fixed in bc05c519.